static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -412,7 +423,11 @@ define([
                         isLoading: false,
                         slot: defineSlot($adSlot)
```

Comment:
What about putting this inside the else branch so we won't run defineSlot if we don't need to.